tests/bluetooth/audio/ascs/src/main.c

```diff
@@ -220,3 +220,79 @@ ZTEST_F(ascs_test_suite, test_abort_client_operation_if_callback_not_registered)
 		      param->response_code);
 	zassert_equal(0x00, param->reason, "unexpected Reason 0x%02x", param->reason);
 }
+
+ZTEST_F(ascs_test_suite, test_release_ase_on_acl_disconnection_client_terminates_cis)
+{
+	const struct bt_gatt_attr *ase;
+	struct bt_bap_stream *stream = &fixture->stream;
+	struct bt_conn *conn = &fixture->conn;
+	struct bt_iso_chan *chan;
+	uint8_t ase_id;
+
+	if (IS_ENABLED(CONFIG_BT_ASCS_ASE_SNK)) {
+		ase = fixture->ase_snk.attr;
+		ase_id = fixture->ase_snk.id;
+	} else {
+		ase = fixture->ase_src.attr;
+		ase_id = fixture->ase_src.id;
+	}
+
+	zexpect_not_null(ase);
+	zexpect_true(ase_id != 0x00);
+
+	bt_bap_unicast_server_register_cb(&mock_bap_unicast_server_cb);
+
+	/* Set ASE to non-idle state */
+	test_preamble_state_streaming(conn, ase_id, stream, &chan,
+				      !IS_ENABLED(CONFIG_BT_ASCS_ASE_SNK));
```

Comment:
Instead of relying on `IS_ENABLED(CONFIG_BT_ASCS_ASE_SNK)` would it make more sense to test this with
1) Sink stream in CIS only
2) Source stream in CIS only
3) Sink and Source stream in same CIS
?